### PR TITLE
Append to log stops working if log is used in between

### PR DESCRIPTION
**Pull request by MichaelClerx**
See #410 

This PR just shows how to debug the issue. See #410 for details of what's going on

### CODE REVIEW

File: myokit/_sim/cvodesim.c [hunk 1]
```diff
@@ -1002,7 +1002,7 @@ for var in model.variables(deep=True, state=False, bound=False, const=False):
                 Py_XDECREF(ret);
                 if (ret == NULL) {
                     flt = NULL;
-                    PyErr_SetString(PyExc_Exception, "Call to append() failed on logging list.");
+                    //PyErr_SetString(PyExc_Exception, "Call to append() failed on logging list.");
```
Comment (MichaelClerx) [inline myokit/_sim/cvodesim.c:1005]:
```suggestion
                    PyErr_SetString(PyExc_Exception, "Call to append() failed on logging list.");
```

File: myokit/_sim/cvodesim.c [hunk 2]
```diff
@@ -1270,7 +1270,7 @@ sim_step(PyObject *self, PyObject *args)
                     Py_DECREF(flt); flt = NULL;
                     Py_XDECREF(ret);
                     if (ret == NULL) {
-                        PyErr_SetString(PyExc_Exception, "Call to append() failed on logging list.");
+                        //PyErr_SetString(PyExc_Exception, "Call to append() failed on logging list.");
```
Comment (MichaelClerx) [inline myokit/_sim/cvodesim.c:1273]:
```suggestion
                        PyErr_SetString(PyExc_Exception, "Call to append() failed on logging list.");
```

**Comment by MichaelClerx** [comment]
Will hold off on adding this test until the minimum Python version loads a matplotlib without the issue by default :D
